demos/src/custom-html-full.mustache

```diff
@@ -27,7 +27,7 @@
 
 				<!-- note: update action redirect, see README for more details -->
 				<div class="o-cookie-message__action o-cookie-message__action--secondary">
-					<a href="https://cookies.ft.com/preferences/manage-cookies?redirect=#"
+					<a href="https://www.ft.com/preferences/manage-cookies?redirect=#"
```

Comment:
Does it still have a redirect query parameter? I tried setting it to an ft article but no redirect happened

Reply:
Seemingly yes for accepting cookies, no for managing them. I'm happy to leave it as is as there are more users than ft.com and it depends on the implementation of the manage cookies page, a redirect would be ideal. Will nudge the ads/privacy team to ask if there should be a redirect for ft.com